# Time in seconds
# Current in amps
# position, velocity, and acceleration in device units
my_joint_actuator:
  actuator_name: 'test_joint_actuator'
  serial_number: '602084008014'
  operation_mode: 'profile_position'
  clear_faults: true

  motor:
    type: 1
    dc_motor:
      nominal_current: 0.675
      max_output_current: 1.35
      thermal_time_constant: 11.6
      number_of_pole_pairs: 1

  sensor:
    type: 1
    incremental_encoder:
      resolution: 1024
      inverted_polarity: false

  safety:
    max_following_error: 2000
    max_profile_velocity: 10000
    max_acceleration: 15000

  velocity_profile:
    acceleration: 8000
    deceleration: 9000
    max_profile_velocity: 9000
  position_profile:
    velocity: 10000
    acceleration: 8000
    deceleration: 9000
    max_profile_velocity: 9000

joint_2:
  actuator_name: 'joint_actuator_2'
  serial_number: '602084007741'
  operation_mode: 'profile_velocity'
  clear_faults: true

  motor:
    type: 1
    dc_motor:
      nominal_current: 0.675
      max_output_current: 1.35
      thermal_time_constant: 11.6
      number_of_pole_pairs: 1

  sensor:
    type: 1
    incremental_encoder:
      resolution: 1024
      inverted_polarity: false

  safety:
    max_following_error: 2000
    max_profile_velocity: 10000
    max_acceleration: 15000

  velocity_profile:
    acceleration: 8000
    deceleration: 9000
    max_profile_velocity: 9000
